src/app/pcasts/dao/subscriptions_dao.py

```diff
@@ -57,3 +57,7 @@ def delete_subscription(user_id, series_id):
     return db_utils.delete_model(maybe_subscription)
   else:
     raise Exception("Specified subscription does not exist")
+
+def get_user_subscriptions_for_series(user_id, series_ids):
```

Comment:
will the unenforced ordering be a problem here?

Reply:
We just want the subscriptions so that we can create a mapping from series_id to subscription_id that will be referenced later. Therefore the order does not matter